docs/uml-class-diagrams/middleware01/production/MiddlewareGisManagerImplementation/MiddlewareGisManagerImplementation.puml

```diff
@@ -2,12 +2,16 @@
 
 package edu.oakland.production.middleware01 {
     class MiddlewareGisManagerImplementation {
+        - DatabaseGisInterface databaseGisInterface
```

Comment:
You will probably need a private variable to stage the LocationDataPoint in between when you get it and when you get the signal from Middleware_02 to store it in the database.
```suggestion
        - DatabaseGisInterface databaseGisInterface
        - LocationDataPoint locationDataPoint
```